wsmaster/che-core-sql-schema/src/main/resources/che-schema/6.4.0/2__add_k8s_metadata.sql

```diff
@@ -0,0 +1,75 @@
+--
+-- Copyright (c) 2012-2018 Red Hat, Inc.
+-- All rights reserved. This program and the accompanying materials
+-- are made available under the terms of the Eclipse Public License v1.0
+-- which accompanies this distribution, and is available at
+-- http://www.eclipse.org/legal/epl-v10.html
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+-- Runtime ---------------------------------------------------------------------
+CREATE TABLE che_k8s_runtime (
+    workspace_id    VARCHAR(255)    NOT NULL,
+    env_name        VARCHAR(255)    NOT NULL,
+    owner_id        VARCHAR(255)    NOT NULL,
+    namespace       VARCHAR(255)    NOT NULL,
+    status          VARCHAR(255)    NOT NULL,
+
+    PRIMARY KEY (workspace_id)
+);
+-- indexes
+CREATE UNIQUE INDEX index_k8s_runtime_workspace_id ON che_k8s_runtime (workspace_id);
```

Comment:
You already have an index on workspace_id. Instead, you need foreign kay on workspace table.